### DISCUSSION TITLE
Please Consider Web Components

### DISCUSSION THREAD

**Discussion by EisenbergEffect** [editablejs/editable#114] [General]
I saw in the readme that Editable is currently based on React. That's not ideal for folks using other libraries or no library. I also saw that you were considering re-working the rendering engine to use a different approach. I'd like to strongly encourage you to use Web Components and native Web Platform capabilities. Building the editor in this way will enable it to work with any library or framework.

**Comment by big-camel** [comment]
This is a good idea

**Reply by XQ0118** [reply]
use native dom will be better

**Comment by byoungd** [comment]
> use native dom will be better
We are working on it.
